src/components/WalletConnect.vue

```diff
@@ -246,6 +270,14 @@ export default defineComponent({
 				this.walletStore.setNetworkTypes(e.key);
 			}
 		},
+		handleAddressSwitch(e: any) {
+			if (e.key) {
+				const addr = e.key;
+				this.walletAddress = addr;
+				this.setAddress(addr);
+				console.log("Address Switched.");
+			}
+		},
```

Comment:
When the user changes the address, please update the tokens information, i.e available and locked

Reply:
Done